### ISSUE TITLE
Improving Installation Script

### ISSUE THREAD

**Issue by mughal4** [bakape/meguca#935]
"apt-get" needs to be run on sudo on a default install but that causes /meguca to be owned by root during "git clone", which later causes "make" to fail because it can't make directories in /meguca.

"createuser -P meguca" is an interactive prompt requesting password, breaks automatic installation.

Nginx missing from install; Meguca listening on port 80 directly will fail on default Debian installs.

**Comment by bakape** [comment]
That's only for reference, so I don't see a problem.

**Comment by setr** [comment]
Even as a reference, it's worth improving if only to avoid issues being created (as I was just about to). At the very least, this issue is useful as documentation for future initial installations.


With that goal in mind, I just got a new installation of meguca running on a Debian Stretch instance, using the docs/install.sh script as the basis. Biggest issue was:

The master branch currently **does not imply a working release**. The install script should instead point to some working, tagged commit, or a branch that does guarantee a working release.

The same issue with bakape/thumbnailer. 

The master branch at time of installation was non-building, and the latest tagged releases of Meguca/thumbnailer are apparently non-compatible (v4.4.0 and v2.0.1 respectively). I resolved this by randomly rolling back both repos till I found a working build, ending up with commits:
**meguca**: 6a88696854d8940f9a5ccbe0a155f82fb27f2a20
**thumbnailer**: f191a43e9c3c7c9522a67dd81ed1aec9cea0280d

**git pull to update meguca is unsafe** for the same reason. I assume update_deps also pulls the latest, possibly broken, master branch of thumbnailer, as the MakeFile does.
